src/components/Login.tsx

```diff
@@ -0,0 +1,15 @@
+import "./login.css";
+
+interface PropsLogin {
+	login: string;
+}
+
+export const Login = ({ login }: PropsLogin) => {
+	return (
+		<div>
+			<button className=" px-6 pt-1 pb-1 navbar__buttonL bg-white rounded-xl hover:text-white hover:border-2 hover:border-white">
+				{login}
```

Comment:
Pokud komponenta se jmenuje Login, tak proc sem dvas string, ktery rika login? To same plati pro SignIn. Udelej Jeden Button, ktery bude mit dva obecne typy Primary a Action treba, onCLick a children. TO potom pouzijes a nebudes potrebovat tyhled dve komponenty